app/ashlar/serializer_fields.py

```diff
@@ -34,7 +36,7 @@ class JsonSchemaField(JsonBField):
     def to_internal_value(self, value):
         super(JsonSchemaField, self).to_internal_value(value)
         try:
-            jsonschema.Draft4Validator.check_schema(value)
+            SchemaModel.validate_schema(value)
```

Comment:
Hmmm, I actually dislike this as it adds a dependency on ashlar. Ideally I'd like to break these field/serializer classes into a separate library. Is there a sensible way to switch the dependency around?